src/capi.rs

```diff
@@ -623,19 +623,6 @@ fn check_tile_log2(n: Result<usize, ()>) -> Result<usize, ()> {
   }
 }
 
-fn check_frame_size(n: Result<usize, ()>) -> Result<usize, ()> {
```

Comment:
Instead of removing this, you should set the minimum to 1. We still need to make sure that it fits into a u16 and that it's non-zero (and positive).